api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1524,12 +1524,14 @@ def trash_container(self) -> Union[labware.Labware, TrashBin, WasteChute]:
         This is the property used to determine where to drop tips and blow out liquids
         when calling :py:meth:`drop_tip` or :py:meth:`blow_out` without arguments.
 
-        On a Flex running a protocol with API version 2.16 or higher, ``trash_container`` is
-        the first ``TrashBin`` or ``WasteChute`` object loaded in the protocol.
-        On a Flex running a protocol with API version 2.15, ``trash_container`` is
-        a single-well fixed trash labware in slot D3.
-        On a an OT-2, ``trash_container`` is always a single-well fixed trash labware
-        in slot 12.
+        The default value depends on the robot type and API version:
+
+        - On a Flex running a protocol with API version 2.16 and later, the first
+          ``TrashBin`` or ``WasteChute`` object loaded in the protocol.
+        - On a Flex running a protocol with API version 2.15, a single-well fixed trash
+          ``Labware`` in slot D3.
+        - On an OT-2, it is always in slot 12. It is a ``Labware`` in API version 2.15
+          or earlier, or a ``TrashBin`` in API version 2.16 and later.
```

Comment:
I think we can simplify this by referring to `InstrumentContext.fixed_trash`.

Either something like this:

```suggestion
        - :py:obj:`ProtocolContext.fixed_trash`, if it exists.
        - Otherwise, the first thing that you've loaded so far with
          :py:obj:`ProtocolContext.load_trash_bin()`
          or :py:obj:`ProtocolContext.load_waste_chute()`.
```

Or, equivalently:

```suggestion
        - On a Flex running a protocol with API version 2.16 and later, the first
          :py:obj:`TrashBin` or :py:obj:`WasteChute` object loaded in the protocol.
        - Otherwise, :py:obj:`ProtocolContext.fixed_trash`.
```

Also, instead of saying "the first `TrashBin` or `WasteChute` loaded in the protocol," it might be more directly relevant to the reader if we linked to `ProtocolContext.load_trash_bin()` and `ProtocolContext.load_waste_chute()`.

Reply:
So like this? 
```
On a Flex running a protocol with API version 2.16 and later, the object returned 
by the first call to :py:meth:`.load_trash_bin` or :py:meth:`.load_waste_chute`.
```

Reply:
Yep that works. I had "**the first thing you loaded with** :py:meth:`.load_trash_bin` or :py:meth:`.load_waste_chute`" [emphasis added] in my draft if that reads better to you, but your call.